Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
Must be getting late.  I understand the intent but I'm not following the details.

@oderwat would need to OK the changes before I'd be comfortable doing them.